crates/wiggle/test-helpers/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wiggle-test"
-version = "0.19.0"
+version = "0.21.0"
```

Comment:
FWIW this is excluded from publication so it's fine to avoid bumping the version here.